src/components/collapsible-motion/collapsible-motion.story.js

```diff
@@ -1,6 +1,8 @@
+/** @jsx jsx */
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { withKnobs, number, boolean } from '@storybook/addon-knobs';
+import { jsx, css } from '@emotion/core';
```

Comment:
feels weird to have to do this, since CollapsibleMotion is supposed to be part of our API. I'm not sure how else to do this though. There must be a way 🤔